src/action.js

```diff
@@ -26,7 +26,15 @@ async function exportSecrets() {
     const defaultOptions = {
         prefixUrl: vaultUrl,
         headers: {},
-        https: {}
+        https: {},
+        retry: {
+            statusCodes: [
+                ...got.defaults.options.retry.statusCodes,
+                // Vault returns 412 when the token in use hasn't yet been replicated
+                // to the performance replica queried. See issue #332.
+                412,
+            ]
```

Comment:
Should we retry on 500's (except 501) too? This is what the Vault Go API client does via [go-retryablehttp](https://github.com/hashicorp/go-retryablehttp#go-retryablehttp).

Reply:
I believe 500 and a few other 5XX are already included in the defaults for the `got` dependency: https://github.com/sindresorhus/got/blob/HEAD/documentation/7-retry.md#retry

I'd like to give that a test to be sure we're preserving that default properly though.

Reply:
How many times will this retry? Should we set a default `limit`? https://github.com/sindresorhus/got/blob/HEAD/documentation/7-retry.md#limit